### ISSUE TITLE
Git clone should try via https if git protocol fails (usually due to firewalls)

### ISSUE THREAD

**Issue by ravenhall** [tadzik/rakudobrew#69]
$ rakudobrew build moar
Update git reference: rakudo
Cloning into 'rakudo'...
warning: expected SRV RR, found RR type 1
fatal: unable to connect to github.com:
github.com[0: 192.30.252.131]: errno=Connection refused
github.com[1: 192.30.252.131]: errno=Connection refused

Failed running git clone git://github.com/rakudo/rakudo.git rakudo at /home/nwaddell/.rakudobrew/bin/rakudobrew line 57.
        main::run("git clone git://github.com/rakudo/rakudo.git rakudo") called at /home/nwaddell/.rakudobrew/bin/rakudobrew line 400
        main::update_git_reference("rakudo") called at /home/nwaddell/.rakudobrew/bin/rakudobrew line 363
        main::build_impl("moar", undef, "") called at /home/nwaddell/.rakudobrew/bin/rakudobrew line 113

**Comment by ravenhall** [comment]
I see in the code where you can set $GIT_PROTOCOL in the environment, so that is a potential workaround. However, if the git protocol fails, why not use https next automagically?

**Comment by tadzik** [comment]
Yeah, I don't see why not :) Good suggestion, and sorry for keeping you hanging for so long.

**Comment by coke** [comment]
We also (will) support the ssh protocol now (or when the PR gets merged)

**Comment by ravenhall** [comment]
Cool.

**Comment by tadzik** [comment]
Closing as the project is not maintained anymore.
